python-frontend/src/test/java/org/sonar/python/semantic/SymbolTableBuilderTest.java

```diff
@@ -387,16 +391,21 @@ public void importing_parent_after_submodule_2() {
 
     assertThat(symbolByName).containsOnlyKeys("werkzeug");
     SymbolImpl werkzeug = (SymbolImpl) symbolByName.get("werkzeug");
-    assertThat(werkzeug.usages()).extracting(Usage::kind).containsExactly(Usage.Kind.IMPORT, Usage.Kind.IMPORT, Usage.Kind.IMPORT, Usage.Kind.OTHER);
+    assertThat(werkzeug.usages()).extracting(Usage::kind).containsExactly(Usage.Kind.IMPORT, Usage.Kind.IMPORT, Usage.Kind.IMPORT, Usage.Kind.OTHER, Usage.Kind.OTHER);
     Map<String, Symbol> childrenSymbolByName = werkzeug.getChildrenSymbolByName();
-    assertThat(childrenSymbolByName.values()).extracting(Symbol::name).contains("datastructures");
+    assertThat(childrenSymbolByName.values()).extracting(Symbol::name).contains("datastructures", "Response");
```

Comment:
Shouldn't we get here `Request` as well? No need to add it just curious.

Reply:
Yes, I just picked one randomly that we couldn't get from simply importing `werkzeug.datastructures.csp` or `werkzeug.datastructures` to ensure the `import werkzeug` was correctly resolved on top of that.